controlplane/pkg/nsmd/nsmd.go

```diff
@@ -43,6 +43,9 @@ const (
 	NsmdDeleteLocalRegistry = "NSMD_LOCAL_REGISTRY_DELETE"
 	ForwarderTimeout        = 1 * time.Hour
 	NSEAliveTimeout         = 1 * time.Second
+
+	// PreferredRemoteMechanism - remote mechanism will be checked first
+	PreferredRemoteMechanism = "PREFERRED_REMOTE_MECHANISM"
```

Comment:
Why we need two duplicate  constants here and controlplane/pkg/remote/forwarder_service.go